azure-iot-device/azure/iot/device/iothub/models/pnp_translation.py

```diff
@@ -0,0 +1,26 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains utility functions to translate basic IoTHub objects
+into PNP objects
+"""
+
+from .commands import Command
+
+
+def method_request_to_command(method_request):
+    """Given a MethodRequest, returns a Command"""
+    pass
+    # return Command(
+    #     request_id=method_request.request_id,
+    #     component_name="",
+    #     command_name=method_request.name,
+    #     payload=method_request.payload,
+    # )
+
+
+def twin_patch_to_writable_property(twin_patch):
```

Comment:
FWIW, a twin _patch_ will never become what we call a WritableProperty object.  I think what we need here is a function to convert a desired property patch into a ClientProperties object.  Yes, the model names around properties are horrible.  I expect them to change.

Reply:
Yeah, since I've decided to leave this alone for now, consider this is just a placeholder